creator-node/src/fileManager.js

```diff
@@ -20,17 +21,21 @@ const SaveFileForMultihashToFSIPFSFallback = config.get('saveFileForMultihashToF
 
 /**
  * Adds file to IPFS then saves file to disk under /multihash name
+ *
+ * If buffer is a metadata, wait on the ipfs add as Discovery Nodes rely on ipfs to fetch metadata.
```

Comment:
this comment is still relevant but i'd rephrase to something like

"If buffer is metadata, await add to ipfs daemon since discovery node checks ipfs first and benefits from increased availability"